index.js

```diff
@@ -48,7 +37,14 @@ module.exports = function (hostname) {
   var NativeModules
   var PlatformConstants
   var AndroidConstants
-  if (typeof window === 'undefined' || !window.__DEV__ || typeof window.require !== 'function') {
+  if (
+    typeof window === 'undefined' ||
+    !window.__DEV__ ||
+    typeof window.require !== 'function' ||
+    // RN >= 0.56
+    // TODO: Get NativeModules for RN >= 0.56
+    typeof window.require.name === 'metroRequire'
```

Comment:
this isn't correct - `window.require.name === 'metroRequire'`, but `typeof window.require.name === 'string'`

Reply:
@deecewan you're right! Also thanks for the PR #6. 😃